stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/internal/ValidationErrorRenderer.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.ui.message.composer.internal
+
+import android.content.Context
+import android.widget.Toast
+import com.getstream.sdk.chat.utils.MediaStringUtil
+import io.getstream.chat.android.common.state.ValidationError
+import io.getstream.chat.android.ui.R
+
```

Comment:
Doesn't look very meaningful, but want to keep the implementation of `MessageComposerView` as clean as possible.